libact/query_strategies/tests/test_realdata.py

```diff
@@ -0,0 +1,112 @@
+"""
+This module uses real world dataset to test the active learning algorithms.
+Since creating own artificial dataset for test is too time comsuming so we would
+use real world data, fix the random seed, and compare the query sequence each
+active learning algorithm produces.
+"""
```

Comment:
I think line 3-5 can be cancelled since users do not have to know why we use real-world data set. Also because the "too time consuming" is not a good reason.